Services/Infrastructure/OJS.Services.Infrastructure/Constants/CacheConstants.cs

```diff
@@ -5,9 +5,11 @@ public static class CacheConstants
         public const int OneHourInSeconds = 60 * 60;
         public const int OneDayInSeconds = 86400;
         public const int TwoMinutesInSeconds = 120;
+        public const int FiveMinutesInSeconds = 300;
 
         public const string MainContestCategoriesDropDown = "MainContestCategoriesDropDown";
         public const string ContestResults = "Contest_id_{0}official{1}full{2}";
+        public const string ContestParticipants = "Contest_id_{0}";
```

Comment:
This is not a good key name. It does not tell you anything, except is for some contest.
Lets use `:` instead of `_` and remove "id" from the name. See this PR for reference: https://github.com/SoftUni-Internal/OpenJudgeSystem/pull/799/files

Also why we don't have a key for Compete and another for Practice participants count?